src/components/PeopleFilters.tsx

```diff
@@ -29,40 +113,60 @@ export const PeopleFilters = () => {
           <div className="level-left">
             <a
               data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
+              role="presentation"
+              className={cn(
+                'button mr-1',
+                { 'is-info': filterParams.getAll('century').includes('16') },
+              )}
+              onClick={() => handleCenturiesApplying('16')}
```

Comment:
Buttons differ only in value (16, 17 etc.), so you can create an array with the necessary values and walk through it with `map` in order not to duplicate the code

Reply:
Oh, that's smart actually, haven't even thought about that